qiskit/visualization/pulse/matplotlib.py

```diff
@@ -632,10 +644,10 @@ def draw(self, schedule, dt, interp_method, plot_range,
 
         # count numbers of valid waveform
 
-        n_valid_waveform, v_max = self._count_valid_waveforms(output_channels,
-                                                              scale=scale,
-                                                              channels=channels,
-                                                              plot_all=plot_all)
+        n_valid_waveform, scale_dict = self._count_valid_waveforms(output_channels,
```

Comment:
a call returning one of its parameters looks funny to me. Is that intended?

Reply:
argument can include `None` as a scaling value and auto scaled value is inserted in the returned dictionary if the value is `None`. I'll change argument name.